bootstrap.sh

```diff
@@ -1 +1,10 @@
 #!/bin/bash
+
+kind create cluster --config kind
+kubectl apply -f mysql-configmap.yaml
+kubectl apply -f mysql-secret.yaml
+kubectl apply -f pv.yaml
+kubectl apply -f pvc.yaml
+kubectl apply -f statefulset.yaml
+kubectl apply -f secret.yml
+kubectl apply -f deployment.yaml
```

Comment:
Verify that the file names and paths used in the `kubectl apply` commands match the actual files in your directory. For example, `mysql-configmap.yaml` should match the file name exactly.